docs/docs/coverage/language/python.md

```diff
@@ -104,6 +104,8 @@ urllib3==1.26.15
 `requirements.txt` files don't contain information about dependencies used for development.
 Trivy could detect vulnerabilities on the development packages, which not affect your production environment.
 
+By default, Trivy doesn't report development dependencies. Use the `--include-dev-deps` flag to include them.
```

Comment:
Trivy doesn't support `--include-dev-deps` flag for `pip`

Reply:
Fixed https://github.com/aquasecurity/trivy/pull/8134/commits/55fb02f3be0c5a663afc943eb822a0b0a1872ac8

Reply:
> Can use link for Include as in [nodejs page](https://trivy.dev/latest/docs/coverage/language/nodejs/)?
By the way, why are `include/exclude` used to link the table and sections and not the package manager names from the first column?

Reply:
By default, we don't show Dev dependencies. 
This link is intended to draw users attention to this and direct them to the section about the file, where they can see that `--include-dev-deps` flag needs to be used.